src/pages/SigningOfficialConsole.js

```diff
@@ -1,245 +1,149 @@
-              div({ className: twoColumnClass + ' cell-header f-center dataset-color' }, ['Active DARs'])
-            ]),
-
-            hr({ className: 'table-head-separator' }),
-
-            div({ className: 'row no-margin' }, [
-              div({ className: twoColumnClass }, ['Carol Smith']),
-              div({ className: twoColumnClass }, ['csmith1212@jnj.org']),
-              div({ className: twoColumnClass }, ['[Broad], [NIH]']),
-              div({ className: twoColumnClass + ' cell header cancel-color'}, ['[Issue/Remove]']),
-              div({ className: twoColumnClass }, ['PI']),
-              div({ className: oneColumnClass }, ['4'])
-            ]),
-
-            div({ className: 'row no-margin' }, [
-              div({ className: twoColumnClass }, ['Jaime Gonzalez']),
-              div({ className: twoColumnClass }, ['jgonzalez1212@jnj.org']),
-              div({ className: twoColumnClass }, ['[Broad]']),
-              div({ className: twoColumnClass + ' cell header cancel-color'}, ['[Issue/Remove]']),
-              div({ className: twoColumnClass }, ['Lab Staff']),
-              div({ className: oneColumnClass }, ['3'])
-            ]),
-
-            PaginatorBar({
-              name: 'dul',
-              total: this.state.electionsList.dul.filter(this.searchTable(searchDulText)).length,
-              limit: dulLimit,
-              currentPage: this.state.currentDulPage,
-              onPageChange: this.handleDulPageChange,
-              changeHandler: this.handleDulSizeChange
+    init();
+  }, []);
+
+  const handleSearchChangeDars = darSearchHandler(darList, setFilteredDarList, setCurrentDarPage);
+  const handleSearchChangeUsers = userSearchHandler(userList, setFilteredUserList, setCurrentUserPage);
+
+  return (
+    div({style: Styles.PAGE}, [
+      div({ isRendered: !isLoading, style: {display: "flex"}}, [
+        div({style: {...Styles.HEADER_CONTAINER, paddingTop: "3rem", paddingBottom: "2rem"}}, [
+          div({style: Styles.TITLE}, ["Welcome " +signingOfficial+ "!"]),
+          div({style: Object.assign({}, Styles.MEDIUM_DESCRIPTION, {fontSize: '18px'})}, [
+            "Your researchers and their submitted Data Access requests are below. ",
+            a({
+              rel: "noopener noreferrer",
+              href: 'https://broad-duos.zendesk.com/hc/en-us/articles/360060402751-Signing-Official-User-Guide',
+              target: '_blank',
+              id: 'so-console-info-link'},
+            ['Click Here for more information'])
+          ])
+        ])
+      ]),
+      div({style: {borderTop: '1px solid #BABEC1', height: 0}}, []),
+      div({style: {display: 'flex',justifyContent: "space-between"}}, [
+        div({ style: Styles.LEFT_HEADER_SECTION}, [
+          div({style: Styles.ICON_CONTAINER}, [
+            img({
+              id: 'user-icon',
+              src: userIcon,
+              style: Styles.HEADER_IMG
             })
```

Comment:
The styles here stretch the icon quite a bit at 60x60. The icon itself is closer to 49x58. Might be better to leave it as it's natural size and maybe center it in the div. 

## PR:
![Screen Shot 2021-07-08 at 8 57 25 AM](https://user-images.githubusercontent.com/116679/124925580-94eed500-dfca-11eb-9774-15644e4d7aff.png)

## Without sizing:
![Screen Shot 2021-07-08 at 8 57 35 AM](https://user-images.githubusercontent.com/116679/124925602-9e783d00-dfca-11eb-868b-7b7c3e5343da.png)